GithubConnect/Public/branch.ps1

```diff
@@ -1,4 +1,26 @@
-﻿Function List-GithubBranches {
+Function Get-GithubBranches {

```

Comment:
My only thought was that i've not seen many uses of the List Verb - in fact (get-command -verb list).count returns 0 on my system - so I didnt think it would be the most discoverable verb to use for those new to those using the module or PowerShell in general